istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -535,6 +538,7 @@ func allConfigCmd(ctx cli.Context) *cobra.Command {
 	}
 
 	allConfigCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", summaryOutput, "Output format: one of json|yaml|short")
+	allConfigCmd.PersistentFlags().BoolVar(&withHeaders, "with-headers", false, "Print summary with heading")
```

Comment:
nit: this will also keep it consistent with the description used in ztunnelconfig.go (Line 215)

```suggestion
	allConfigCmd.PersistentFlags().BoolVar(&withHeaders, "with-headers", false, "Print summary with headers")
```